src/invoices/Voucher/VoucherEditForm/VoucherEditContainer.js

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { get } from 'lodash';
+
+import { Paneset } from '@folio/stripes/components';
+import {
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import { LoadingPane } from '../../../common/components';
+import {
+  invoiceResource,
+  VOUCHER_BY_ID,
+  configVoucherNumber,
+} from '../../../common/resources';
+import { getIsAllowVoucherNumberEdit } from '../utils';
+import VoucherEditForm from './VoucherEditForm';
+
+const VoucherEditContainer = ({ match: { params }, mutator, resources }) => {
+  const [voucher, setVoucher] = useState();
+  const vendorInvoiceNo = get(resources, ['invoice', 'records', 0, 'vendorInvoiceNo']);
+  const isAllowVoucherNumberEdit = getIsAllowVoucherNumberEdit(get(resources, ['configVoucherNumber', 'records', 0]));
+
+  useEffect(() => {
+    mutator.voucher.GET({
+      params: {
+        query: `voucherId=${params.voucherId}`,

```

Comment:
there is no GET by id API? https://s3.amazonaws.com/foliodocs/api/mod-invoice/voucher.html#voucher_vouchers__id__get